### PR TITLE
Moved last set of Tutorials

### PR DESCRIPTION
**Pull request by alwasega**
I have moved the last set of docs marked for the `Tutorials` section of the repository. These are "basics" files, which @minrk  suggested [here](https://github.com/jupyterhub/jupyterhub/issues/4272#issuecomment-1380297090) that we could mark as Tutorials for now but can be later rewritten to comply with the Diataxis definition of tutorials.

### CODE REVIEW

**Comment by alwasega** [comment]
Please check this @sgibson91.

File: docs/source/tutorial/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Tutorials
 
-This section of the documentation provides step-by-step tutorials to help you achieve a specific goal. The tutorials should be a good place to start learning about JupyterHub and how it works.
+The _Tutorials_ provide step-by-step lessons to help you achieve a specific goal. The tutorials should be a good place to start learning about JupyterHub and how it works.
```
Comment (sgibson91) [inline docs/source/tutorial/index.md:3]:
```suggestion
_Tutorials_ provide step-by-step lessons to help you achieve a specific goal. They should be a good place to start learning about JupyterHub and how it works.
```

**Review by sgibson91 — APPROVED** [review]
LGTM!

It would be great to start documenting which files need rewriting in a new issue, in case we don't have enough time before the end of the internship.

**Comment by alwasega** [comment]
> It would be great to start documenting which files need rewriting in a new issue, in case we don't have enough time before the end of the internship.

Thanks @sgibson91. Sure, I will create a new issue documenting this.

**Pull request merged**